src/extensions/hooks/adopted-probe/src/validate-fields.ts

```
@@ -56,39 +56,41 @@ export const validateTags = async (fields: Fields, keys: string[], accountabilit
 	const response = await axios<{totalResultsCount: number, geonames: City[]}>(url, {
 		timeout: 5000,
 	});
 
 	const cities = response.data.geonames;
 
 	if (cities.length === 0) {
-		throw payloadError('No valid cities found. Please check "city" value. City search algorithm can be checked here: https://www.geonames.org/advanced-search.html?featureClass=P');
+		throw payloadError('No valid cities found. Please check "city" and "country" values. Search algorithm can be checked here: https://www.geonames.org/advanced-search.html?featureClass=P');
 	}
 
 	const city = cities[0]!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)